src/utils/notify/index.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable */
+const admin = require('firebase-admin');
+const { query: Hasura } = require('../hasura');
+const { getUser, getFcmToken } = require('./queries/queries');
+const constructNotificationMessage = require('./constructNotificationMessage');
+const constructNotificationBody = require('./constructNotificationBody');
+
+const notify = async (entityTypeId, entityId, actorId, notifierIds) => {
+  try {
+    const response = await Hasura(getFcmToken, {
```

Comment:
Can you import the notify.deprecated.js and call it's notify method here & forward these same parameters to it?